web/src/main/java/org/mskcc/cbio/oncokb/api/pub/v1/TrialsApiController.java

```diff
@@ -64,14 +66,19 @@ public ResponseEntity<List<Trial>> trialsMatchingGet(
         AmazonS3 s3client = AmazonS3ClientBuilder.standard()
             .withCredentials(new AWSStaticCredentialsProvider(credentials)).withRegion(s3Region).build();
 
-        S3Object s3object = s3client.getObject("oncokb", "drug-matching/result.json");
-        S3ObjectInputStream inputStream = s3object.getObjectContent();
+        S3Object s3objectTrials = s3client.getObject("oncokb-clinical-trials", "precision_med_trial_data.json");
+        S3ObjectInputStream inputStreamTrials = s3objectTrials.getObjectContent();
         JSONParser jsonParser = new JSONParser();
-        JSONObject jsonObject = (JSONObject) jsonParser.parse(new InputStreamReader(inputStream, "UTF-8"));
+        JSONObject jsonObjectTrials = (JSONObject) jsonParser.parse(new InputStreamReader(inputStreamTrials, "UTF-8"));
+        
+        S3Object s3objectOncotree = s3client.getObject("oncokb-clinical-trials", "oncotree_to_trials_precision_med.json");
```

Comment:
The naming of these files should be standardized. `precision_med` should be prefix or postfix, but not the mix usage.

Reply:
also the name `oncotree_to_trials` does not make sense

Reply:
Yes that was mainly for my testing purposes. I will update the names